internal/wspace/finder.go

```diff
@@ -19,23 +19,39 @@ import (
 	"strings"
 )
 
-const workspaceFile = "WORKSPACE"
+var workspaceFiles = []string{"WORKSPACE.bazel", "WORKSPACE"}
 
-// Find searches from the given dir and up for the WORKSPACE file
+// FindWorkspaceFile returns a path to a file in the provided root directory,
+// either to an existing WORKSPACE or WORKSPACE.bazel file, or to root/WORKSPACE
+// by default. Note that this function does NOT recursively check parent directories
```

Comment:
In stead of "by default", say "if neither file exists".

Also, add a period at the end of the last sentence.